openlibrary/catalog/get_ia.py

```diff
@@ -79,9 +82,13 @@ def get_marc_record_from_ia(identifier):
     # Try marc.xml first
     if marc_xml_filename in filenames:
         data = urlopen_keep_trying(item_base + marc_xml_filename).read()
-        if data[:10].find('<?xml') != -1:
+        try:
             root = etree.fromstring(data)
-            return MarcXml(root)
+            if root.tag == "{http://www.loc.gov/MARC21/slim}record":
```

Comment:
It seems like this file would fail the above check: https://ia800608.us.archive.org/17/items/windhaven00geor_0/windhaven00geor_0_marc.xml

Or is this file malformatted?

Reply:
Very good catch, thank you @cdrini. `collection` is a valid top-level collection of `record`s.  `collection` and `record` are the only two valid top-level elements. I'll update the code and tests tomorrow so that they are both accepted!

Reply:
:+1: Got lucky on that one; that happened to be the marcxml for the first book on openlibrary.org the day I did this CR :P